human-mutation.csl

```diff
@@ -13,11 +13,11 @@
     <category field="medicine"/>
     <issn>1059-7794</issn>
     <eissn>1098-1004</eissn>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2016-12-27T20:07:23+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <macro name="author-short">
-    <names variable="author">
+    <names variable="author" suffix=",">
```

Comment:
great. Thank you for the explanation. I'll keep an eye on these sort of things from now on.